src/node_crypto.cc

```diff
@@ -751,6 +751,23 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
   CHECK_EQ(sc->ca_store_, nullptr);
 
   if (!root_cert_store) {
+#if defined(NODE_SYSTEM_CERT_STORE)
+    // *Assume* OpenSSL is setup correctly, which is the case
+    // for distribution supplied versions.
```

Comment:
Actually, this does not appear to be the case. Look in `openssl.gypi`. I've tested this with OpenSUSE Leap 42.1 which needs bundled OpenSSL and it certainly uses the /etc/ssl path as defined in the openssl.gypi file.